colossalai/kernel/triton/context_attn_unpad.py

```diff
@@ -185,6 +185,184 @@ def _fwd_context_paged_attention_kernel(
     return
 
 
+# Triton 2.1.0
+# TODO(yuanheng-zhao): This is a temporary dispatch to use the new layout for kcache
+# merge `_fwd_context_paged_attention_kernel_v2` with `_fwd_context_paged_attention_kernel` later
+# as the kcache layout has been supported in the whole triton flow.
+@triton.jit
+def _fwd_context_paged_attention_kernel_v2(
+    Q,
+    K,
+    V,
+    O,
+    KCache,  # [num_blocks, num_kv_heads, block_size, head_dim]
+    VCache,  # [num_blocks, num_kv_heads, head_dim // x, block_size, x]
```

Comment:
The shapes of the key cache and value cache should be swapped.

Reply:
Revised